core/bootstrap/src/main/java/org/wildfly/swarm/bootstrap/modules/ContainerModuleFinder.java

```diff
@@ -89,6 +89,40 @@ public void buildModule(ModuleSpec.Builder builder, ModuleLoader delegateLoader)
                         null,
                         ModuleIdentifier.create("javax.enterprise.api"), false));
 
+        builder.addDependency(
```

Comment:
Does adding these to the container module prevent a user from overriding them, ie. by providing their own implementation in a WAR?

Might be worth adding a test to verify that works